src/adagrams.js

```diff
@@ -1,15 +1,151 @@
+
+// Implement this method for wave 1
 export const drawLetters = () => {
-  // Implement this method for wave 1
+  const allLetters = []
+  const LETTER_POOL = {
```

Comment:
It would be great to have this created outside of the function so it does not have to be recreated each time the function is called.